lib/api/graph.js

```diff
@@ -43,8 +45,63 @@ function snapshot(query, from, to, eventQuery, callback){
     client.request("GET", "/graph/snapshot", params, callback);
 }
 
+/*section: graph
+ *comment: create an embed graph of a metric query
+ *params:
+ *  graph_json: The request array to pass create in the embed
+ *  timeframe: (1_hour, 4_hours, 1_day, 2_days, and 1_week)
+ *  size: (small, medium, large, xlarge)
+ *  legend: yes or no
+ *  title: The title of the embed
+ *  callback: function(err, res)
+ *example: |
+ *  ```javascript
+ *  var dogapi = require("dogapi");
+ *  var options = {
+ *    api_key: "api_key",
+ *    app_key: "app_key"
+ *  };
+ *  dogapi.initialize(options);
+ *  var query = "system.cpu.idle{*}";
+ *  var graph_json = {
+ *         "viz": "timeseries",
+ *         "requests": [
+ *           {
+ *             "q": "system.cpu.idle{*}",
+ *             "aggregator": "avg",
+ *             "conditional_formats": [],
+ *             "type": "area"
+ *           }
+ *         ]
+ *       }
+ *  var timeframe = '1_hour';
+ *  var size = 'xlarge';
+ *  var legend = 'yes';
+ *  var title = 'my awesome embed';
+ *  dogapi.graph.embed(graph_json, timeframe, size, legend, title, function(err, res){
+ *    console.dir(res);
+ *  });
+ *  ```
+ */
+function createEmbed(graph_json, timeframe, size, legend, title, callback){
+    var lastArgument = arguments.length -1;
+
+    var params = {
+        body: 'graph_json=' + JSON.stringify(graph_json) +
```

Comment:
data must be url form encoded. DataDog does not really make this obvious. I ran into a few issues figuring it out.